packages/bpk-tokens/src/react-native-output-test.js

```diff
@@ -24,14 +24,6 @@ import androidTokens from '../tokens/base.raw.android.json';
 import iosTokens from '../tokens/base.raw.ios.json';
 
 const ANDROID_ONLY_PROPS = [
-  'BORDER_RADIUS_LG',
```

Comment:
These are still needed on Android, why have they gone missing?

Reply:
Because they're not `ANDROID_ONLY` anymore.
This is a list of all the values that are only on Android. As these are available on both again, they're not in the list.